test/data.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Make sure that the src/data/*.json files respect the right JSON schema
+ */
+
+const assert = require("assert");
+const schema = require("../schema/data.json");
```

Comment:
schema/data.json should be part of this PR :)

Reply:
It should :)